### PR TITLE
[MWAN] Fitelnet tutorial

### PR DESCRIPTION
**Pull request by marciocloudflare**
Closes PCX-7844

### CODE REVIEW

File: content/magic-wan/third-party/fitelnet.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+These configurations were tested on FITELnet F220 and F70 series with the following firmware versions:
+  - **F220 series**: Version 01.11(00)
+  - **F70 series**: Version 01.09(00)
+
+## IPsec configuration
+
+### Magic WAN configuration
+
+1. Go to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account.
+2. Go to to **Magic WAN** > **Manage Magic WAN Configuration** > **Configure**.
```
Comment (jason-cf) [inline content/magic-wan/third-party/fitelnet.md]:
```suggestion
2. Go to **Magic WAN** > **Manage Magic WAN Configuration** > **Configure**.
```

File: content/magic-wan/third-party/fitelnet.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+3. From the **Tunnels** tab, select **Create**.
+4. For the first IPsec tunnel, ensure the following settings are defined. ([settings not mentioned here](/magic-wan/get-started/configure-tunnels/#add-tunnels) can be left with their default settings):
```
Comment (jason-cf) [inline content/magic-wan/third-party/fitelnet.md]: can this be reworded to something like: "optional configurations can be found here" ?

File: content/magic-wan/third-party/fitelnet.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+### FITELnet router configuration
+
+#### Router 1 settings
+
+Use the CLI to configure these settings:
+
+```txt
+interface Tunnel 1
+ ip address 10.0.0.0 255.255.255.254
```
Comment (jason-cf) [inline content/magic-wan/third-party/fitelnet.md:44]:
```suggestion
 ip address 10.0.0.1 255.255.255.254
```

File: content/magic-wan/third-party/fitelnet.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+exit
+!
+```
+
+#### Router 2 settings
+
+Use the CLI to configure these settings:
+
+```txt
+interface Tunnel 1
```
Comment (jason-cf) [inline content/magic-wan/third-party/fitelnet.md]:
```suggestion
interface Tunnel 2
```

**Review by jason-cf — APPROVED** [review]

File: content/magic-wan/third-party/fitelnet.md (hunk 3 above)
Comment (marciocloudflare) [reply content/magic-wan/third-party/fitelnet.md:44]: On the router side the correct IP is the one I had

**Pull request merged**